src/app/clusters/on-off-server/on-off-server.cpp

```diff
@@ -49,6 +52,42 @@ OnOffServer OnOffServer::instance;
 
 static OnOffEffect * inst(EndpointId endpoint);
 
+/**********************************************************
+ * Matter timer scheduling glue logic
+ *********************************************************/
+
+void OnOffServer::timerCallback(System::Layer *, void * callbackContext)
+{
+    auto control = static_cast<EmberEventControl *>(callbackContext);
+    (control->callback)(control->endpoint);
```

Comment:
Hmm. If all these functions still refer to `EmberEventControl`, would it make sense to implement them in a single place and reuse?

Reply:
Ah, I see. That's fine with me. I was just curious.